File: fix_categories.py
import json

with open('src/content/projects.json', 'r') as f:
    data = json.load(f)

data[1]['category'] = "AI Agents"
data[3]['category'] = "WORKFLOW AUTOMATION"

with open('src/content/projects.json', 'w') as f:
    json.dump(data, f, indent=2)

File: fix_categories.js
const fs = require('fs');
const data = JSON.parse(fs.readFileSync('src/content/projects.json', 'utf8'));
data[1].category = "AI Agents";
data[3].category = "WORKFLOW AUTOMATION";
fs.writeFileSync('src/content/projects.json', JSON.stringify(data, null, 2));
